lib/IRGen/GenBuiltin.cpp

```diff
@@ -120,6 +120,12 @@ getLoweredTypeAndTypeInfo(IRGenModule &IGM, Type unloweredType) {
   return {lowered, IGM.getTypeInfo(lowered)};
 }
 
+static std::pair<SILType, const TypeInfo &>
+getMaximallyAbstractedLoweredTypeAndTypeInfo(IRGenModule &IGM, Type unloweredType) {
```

Comment:
Is this function unused?

Reply:
Good catch!
And also the test was wrong: I need to disable the target-constant-folding pass. Otherwise the builtins are already constant folded before they reach IRGen.
Fixed.